resources/public/api/conf/1.0/benefit_unignore.yaml

```diff
@@ -5,33 +5,14 @@ description: |
   A National Insurance number, tax year and benefit ID must be provided.
   
   ### Test data
-  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
-  <table>
-      <thead>
-          <tr>
-              <th>Header Value (Gov-Test-Scenario)</th>
-              <th>Scenario</th>
-          </tr>
-      </thead>
-      <tbody>
-          <tr>
-              <td><p>N/A - DEFAULT</p></td>
-              <td><p>Simulates success response.</p></td>
-          </tr>
-          <tr>
-              <td><p>UNIGNORE_FORBIDDEN</p></td>
-              <td><p>Simulates the scenario in which a benefit is customer added and therefore cannot be unignored.</p></td>
-          </tr>
-          <tr>
-              <td><p>NOT_FOUND</p></td>
-              <td><p>Simulates the scenario where no data is found.</p></td>
-          </tr>
-          <tr>
-              <td><p>TAX_YEAR_NOT_ENDED</p></td>
-              <td><p>Simulates the scenario where the tax year provided has not ended.</p></td>
-          </tr>
-      </tbody>
-  </table>
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario) | Scenario                                                                                          |
+  |----------------------------------|---------------------------------------------------------------------------------------------------|
+  | N/A - DEFAULT                    | Simulates success response.                                                                       |
+  | UNIGNORE_FORBIDDEN               | Simulates the scenario in which a benefit is customer added and therefore cannot be unignored.    |
```

Comment:
I think we could slightly reduce the width of this table as the tables should only be as wide as they need to be